Makefile

```diff
@@ -79,7 +80,7 @@ battletest: ## Run randomized, racing, code-covered tests
 		-tags random_test_delay
 
 e2etests: ## Run the e2e suite against your local cluster
-	cd test && CLUSTER_NAME=${CLUSTER_NAME} go test \
+	cd test && GIT_REF=${GIT_REF} CLUSTER_NAME=${CLUSTER_NAME} go test \
```

Comment:
I guess this could be wrong if someone isn't running in the checked out directory as their applied image

Reply:
Yeah, I think the main use-case here is to pass this through during automation. I'm not sure if it's necessarily required to have a good local dev story for these metrics beyond being able to test that they are firing reasonable values.